mhs/inbound/inbound/request/handler.py

```diff
@@ -46,6 +46,9 @@ def initialize(self, workflows: Dict[str, workflow.CommonWorkflow],
         else:
             request_message = self._extract_incoming_sync_request_message()
 
+        interaction_id = request_message.message_dictionary[ebxml_envelope.ACTION]
```

Comment:
@colin-bateson Reminder that we still need to figure out the sync message stuff, if that should be in the inbound handler or not. If it should be, then that potentially could have an impact on this line.